src/EditorFeatures/CSharpTest2/Recommendations/DynamicKeywordRecommenderTests.cs

```diff
@@ -633,5 +633,14 @@ public async Task TestAfterAsync()
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestNotAfterAsyncAsType()
             => await VerifyAbsenceAsync(@"class c { async async $$ }");
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestInFunctionPointerType()
+        {
+            await VerifyKeywordAsync(@"
+class C
+{
+    delegate*<$$");
```

Comment:
presumably this isn't actually allowed?  or is it?  wonky!

Reply:
Of course it's allowed. Would kinda suck if you couldn't nest function pointers.

Reply:
This is 'dynamic'. I'm surprised you can have a funcptr that takes a dynamic arg.

Reply:
You can't convert a function pointer to `dynamic`. But a function pointer is just a method signature. You can put pretty much anything you can put in a normal method signature in a function pointer.